From a8a1eef5e07a104df457a92d054d3b860fd5db27 Mon Sep 17 00:00:00 2001
From: Aleix Pol <aleixpol@kde.org>
Date: Tue, 9 Jan 2024 01:39:32 +0100
Subject: [PATCH] Use the right type for the sequential value

We were losing precision as soon as we copied it over
---
 src/pipewiresourcestream.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pipewiresourcestream.h b/src/pipewiresourcestream.h
index 1327a3b..8db9943 100644
--- a/src/pipewiresourcestream.h
+++ b/src/pipewiresourcestream.h
@@ -62,7 +62,7 @@ Q_DECLARE_METATYPE(PipeWireCursor);
 
 struct KPIPEWIRE_EXPORT PipeWireFrame {
     spa_video_format format;
-    std::optional<int> sequential;
+    std::optional<quint64> sequential;
     std::optional<std::chrono::nanoseconds> presentationTimestamp;
     std::optional<DmaBufAttributes> dmabuf;
     std::optional<QImage> image;
-- 
GitLab

